.gitattributes

```diff
@@ -1,3 +1,3 @@
 * text=auto
-
+*.map text eol=lf
```

Comment:
It just need `text` to be set, c.f. https://git-scm.com/docs/gitattributes see the `Set` section for `text`.

```suggestion
*.map text
```

Reply:
https://git-scm.com/docs/gitattributes#_text